decisions/wc_dark_cult_decisions.txt

```diff
@@ -235,7 +237,21 @@ decisions = {
 						home = PREV
 						owner = THIS
 						troops = {
-							demonicshock = { 500 500 }
+							heavy_infantry = { 600 600 }
+							light_infantry = { 100 100 }
+							war_machines = { 100 100 }
+						}
+						attrition = 0
+						scaled_by_biggest_garrison = 1
+						reinforces = no
+						maintenance = no
+					}
+					spawn_unit = {
+						province = PREV
+						home = PREV
+						owner = THIS
+						troops = {
+							elite_infantry = { 50 50 }
```

Comment:
I don't think this is ok

<details>

![image](https://user-images.githubusercontent.com/46576860/62796316-9dd68380-bae1-11e9-853c-71e4e2ad23f9.png)
![image](https://user-images.githubusercontent.com/46576860/62796326-a202a100-bae1-11e9-9925-2a7effc63b89.png)

</details>